src/components/UI/GameList.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useContext, useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import { useTranslation } from 'react-i18next'
+
+import ContextProvider from '../../state/ContextProvider'
+import { GameStatus } from '../../types'
+import { getProgress } from '../../helper'
+const { ipcRenderer } = window.require('electron')
+interface List {
+    cover: string
+    logo: string
+    title: string
+    appName: string
+    isInstalled: boolean
+}
+
+interface InstallProgress {
+    percent: string
+    bytes: string
+    eta: string
+}
+
+const GameList = ({ cover, title, appName, isInstalled }: List) => {
+    const [progress, setProgress] = useState({
+        percent: '0.00%',
+        bytes: '0/0MB',
+        eta: '',
+    } as InstallProgress)
+    const { t } = useTranslation()
+
+    const { libraryStatus } = useContext(ContextProvider)
+
+    const gameStatus: GameStatus = libraryStatus.filter(
```

Comment:
I'll about a different way of storing this info since now we are repeating it in two different places.